wiki/docs/tablet-satellite/android-config/microphone.md

```diff
@@ -23,18 +23,18 @@ Detailed install video: https://youtu.be/-7cHfqZMa1c
 
 ### Configuration
 1. Find the IPWebcam Icon and tap it to launch.
-1. Scroll down until you see the *IP Webcam settings* section.
-1. Tap on **Power management** and under *Misc* Turn on **Disable notification** and **Stream on device boot**. 
+2. Scroll down until you see the **Service Control** heading and tap on **Optional Permissions**.
+1. Tap **Allow streaming in background** and then turn on **Allow display over other apps**.
+1. Tap back and then activate the checkbox on **Show camera overlay**. This will prevent another window from appearing over your dashboard.
+1. Tap back and scroll up until you see **Power management** tap it and under *Misc* Turn on **Disable notification** and **Stream on device boot**. 
```

Comment:
Why not put this closer to start so you don't scroll down then back up?  I don't see where order matters much on these as long as the settings are set.  Is there a reason?